seqr/views/apis/dataset_api.py

```diff
@@ -110,6 +112,37 @@ def add_variants_dataset_handler(request, project_guid):
     Family.bulk_update(
         request.user, {'analysis_status': Family.ANALYSIS_STATUS_ANALYSIS_IN_PROGRESS}, guid__in=family_guids_to_update)
 
+    if project.workspace_name and project.workspace_namespace:
+        if has_analyst_access(project):
+            safe_post_to_slack(
+                SEQR_SLACK_DATA_ALERTS_NOTIFICATION_CHANNEL,
+                """{num_sample} new samples are loaded in https://seqr.broadinstitute.org/project/{guid}/project_page
+                ```{samples}```
+                """.format(
+                    num_sample=len(matched_sample_id_to_sample_record),
```

Comment:
so this is not the number of new samples that were added, this is the number of total samples that were added. We need to let them know which samples were added that never had a record in seqr before

Reply:
So, I'm going to update `match_sample_ids_to_sample_records()` to return a list of new samples.